Changelog.md

```diff
@@ -14,7 +14,8 @@ Compiler Features:
  * SMTChecker: Properties that are proved safe are now reported explicitly at the end of the analysis. By default, only the number of safe properties is shown. The CLI option ``--model-checker-show-proved-safe`` and the JSON option ``settings.modelChecker.showProvedSafe`` can be enabled to show the full list of safe properties.
  * SMTChecker: Group all messages about unsupported language features in a single warning. The CLI option ``--model-checker-show-unsupported`` and the JSON option ``settings.modelChecker.showUnsupported`` can be enabled to show the full list.
  * Yul EVM Code Transform: If available, use ``push0`` instead of ``codesize`` to produce an arbitrary value on stack in order to create equal stack heights between branches.
-
+ * Commandline Interface: Add ``--ast-compact-json`` option with Yul input (e.g. with ``--strict-assembly``, ``--yul``)
```

Comment:
Just a note: you're gonna have to rebase and move these entries up to the `0.8.21` section, since we released a couple of days ago. Also, the entries should be alphabetically sorted.